weaknesses/MASVS-NETWORK/MASWE-0051.md

```diff
@@ -19,3 +19,31 @@ status: draft
 
 ---
 
+## Overview
+
+Applications that open network ports without proper protection are vulnerable to unauthorized access and potential exploitation. This weakness occurs when an application listens on a network port and accepts incoming connections without adequate security measures, allowing other applications or attackers to connect and interact with it.
```

Comment:
Interesting. I was just spinning up a test app and it works on an Android 13 device, I could connect via netcat from my macOS. I thought this is not possible in more recent Android versions to just open a network port in the App. Would this go through the app review process from iOS or Android? 
 
![image](https://github.com/user-attachments/assets/b2967de4-44ef-4bd9-a6c9-75cf3d943feb)